packages/components/src/text-area/src/TextArea.css

```diff
@@ -1,14 +1,14 @@
 .o-ui-text-area {
-    font-size: var(--o-ui-fs-3);
+    font-size: var(--hop-body-md-font-size);
 }
 
 .o-ui-text-area textarea {
-    --o-ui-textarea-padding: var(--o-ui-sp-3);
+    --o-ui-textarea-padding: var(--hop-space-inset-squish-md);
     resize: vertical;
     /* Fixes a bug where Firefox adds one more row than the one specified. */
     overflow-x: hidden;
-    padding: var(--o-ui-sp-2) var(--o-ui-textarea-padding);
-    min-height: var(--o-ui-sz-11);
+    padding: var(--hop-space-inset-sm) var(--o-ui-textarea-padding);
```

Comment:
You should only use squish and stretch as a shorthand. Otherwise this won't make sense.